packages/manager/modules/pci/src/projects/onboarding/onboarding.constant.js

```diff
@@ -0,0 +1,16 @@
+export const TRUSTED_CUSTOMER_CALLBACK_PHONE = {
+  DEFAULT: 'https://www.ovhcloud.com/en-gb/enterprise/trusted-zone/#callback',
+  FR: 'https://www.ovhcloud.com/fr/enterprise/trusted-zone/#callback',
+  DE: 'https://www.ovhcloud.com/de/enterprise/trusted-zone/#callback',
+  ES: 'https://www.ovhcloud.com/es-es/enterprise/trusted-zone/#callback',
+  IE: 'https://www.ovhcloud.com/en-ie/enterprise/trusted-zone/#callback',
+  IT: 'https://www.ovhcloud.com/it/enterprise/trusted-zone/#callback',
+  NL: 'https://www.ovhcloud.com/nl/enterprise/trusted-zone/#callback',
+  PL: 'https://www.ovhcloud.com/pl/enterprise/trusted-zone/#callback',
+  PT: 'https://www.ovhcloud.com/pt/enterprise/trusted-zone/#callback',
+  GB: 'https://www.ovhcloud.com/en-gb/enterprise/trusted-zone/#callback',
```

Comment:
```suggestion

```
GB not required. DEFAULT will take care of it.

Reply:
Yes sure, i preferred to have the dedicated entry.
I use default one for all missed others subs that could be forgot.